File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1034.cs
using System;

class URI
{
    static void Main() {
        int e, alc=0, gas=0, die=0;
        while(true){
            e = int.Parse(System.Console.ReadLine().Trim());
            if(e==4){
                break;
            } else if(e==1){
                alc++;
            } else if(e==2){
                gas++;
            } else if(e==3){
                die++;
            }
        }
        Console.Write("MUITO OBRIGADO\n");
        Console.Write("Alcool: {0}\n", alc);
        Console.Write("Gasolina: {0}\n", gas);
        Console.Write("Diesel: {0}\n", die);
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1034.lua
alc=0
gas=0
die=0

while(true) do
    e = io.read("*number")
    if(e==4) then
        break
    elseif(e==1) then
        alc=alc+1
    elseif(e==2) then
        gas=gas+1
    elseif(e==3) then
        die=die+1
    end
end
print("MUITO OBRIGADO")
print("Alcool: "..alc)
print("Gasolina: "..gas)
print("Diesel: "..die)